cli/hzn.go

```diff
@@ -363,6 +366,20 @@ Environment Variables:
 	exNodeRemovePolicyIdTok := exNodeRemovePolicyCmd.Flag("node-id-tok", msgPrinter.Sprintf("The Horizon Exchange node ID and token to be used as credentials to query and modify the node resources if -u flag is not specified. HZN_EXCHANGE_NODE_AUTH will be used as a default for -n. If you don't prepend it with the node's org, it will automatically be prepended with the -o value.")).Short('n').PlaceHolder("ID:TOK").String()
 	exNodeRemovePolicyNode := exNodeRemovePolicyCmd.Arg("node", msgPrinter.Sprintf("Remove policy for this node.")).Required().String()
 	exNodeRemovePolicyForce := exNodeRemovePolicyCmd.Flag("force", msgPrinter.Sprintf("Skip the 'are you sure?' prompt.")).Short('f').Bool()
+	exNodeServiceConfigStateCmd := exNodeCmd.Command("serviceconfigstate", "List and manage service config state in open horizon exchange.")
+	exNodeServiceConfigStateListCmd := exNodeServiceConfigStateCmd.Command("list", "Shows list service config states of node in open horizon exchange.")
+	exNodeServiceConfigStateListNode := exNodeServiceConfigStateListCmd.Arg("node", "Service config state list of this node.").Required().String()
+	exNodeServiceConfigStateListIdTok := exNodeServiceConfigStateListCmd.Flag("node-id-tok", msgPrinter.Sprintf("The Horizon Exchange node ID and token to be used as credentials to query and modify the node resources if -u flag is not specified. HZN_EXCHANGE_NODE_AUTH will be used as a default for -n. If you don't prepend it with the node's org, it will automatically be prepended with the -o value.")).Short('n').PlaceHolder("ID:TOK").String()
```

Comment:
I dont think we need this. In practice, node tokens are generated by the node itself during registration, so no human should ever know a node token. This applies to all 3 instances of this flag.